wordle-web/src/scripts/__tests__/wordsService.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { describe, it, expect } from 'vitest'
+import { WordsService } from '../wordsService'
+
+describe('ValidWords', () => {
```

Comment:
File name could maybe be more specific so you know it's relative to the validWords function. Or since I believe that's just what they wanted the method named (not the component), you could instead name the component wordsService or something.

Reply:
The convention for naming test files seems like to me it's (fileName).specs.ts and any tests related to that file go there.